Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Use Node.js 18 as the base image
+FROM node:20.8.0-alpine
```

Comment:
about dockerfile base image.
please use `alpine:3.16`.
opstack using this as follow
https://github.com/oasysgames/oasys-opstack/blob/develop/ops/docker/Dockerfile.packages#L16